test/lld/duplicate_imports.wast.out

```diff
@@ -101,4 +101,39 @@
   )
  )
 )
-;; METADATA: { "asmConsts": {},"staticBump": 13, "initializers": ["__post_instantiate"], "declares": ["puts"], "externs": [], "implementedFunctions": ["___post_instantiate","_main","_stackSave","_stackAlloc","_stackRestore","___growWasmMemory"], "exports": ["memory","__post_instantiate","main","__heap_base","__data_end","stackSave","stackAlloc","stackRestore","__growWasmMemory"], "invokeFuncs": ["invoke_ffd"] }
+(;--BEGIN METADATA --
+{
+"staticBump": 13,
```

Comment:
nit: I feel this would be nicer to indent everything an additional tab (aka 2 spaces), but this is probably just me